📨 Feature/modal-contact: Contact Modal

GreenieWarren opened the pull request:
Este es mi primer pull request. No mas aprendiendo acá lets goooo

GreenieWarren committed:
Pequeños cambios, preparando carpetas

GreenieWarren committed:
Cambio de nombre de carpetas

Haciendo un commit antes de hacer algo que la cague xd

GreenieWarren committed:
Me falto este

GreenieWarren committed:
Modal Abrido Hehe

LETS GOO Y ENCAPSULAO PAPA

Hector1567XD committed:
Merge branch 'improvement/edit-profile-styles' into feature/modal-contact

Hector1567XD committed:
Solving Greenie existencial Conflicts 🗿

GreenieWarren committed:
Estilizando el modal

Ahora si vi como conectar propiamente el classname, gracias Anna tqm <3

GreenieWarren committed:
Cambio de interfaces

Además de eso, puse mas estilizados y encapsulé dentro de lo que pude, pero veo oportunidades de mejora en las que trabajaré luego

GreenieWarren committed:
Creación de hooks

Cree varias carpetas nuevas y cambie el nombre de las que había creado para que entraran dentro del estandar

GreenieWarren committed:
GÜATAFAC SE ESTROPEARON SKILLS AYUDA

AYUDAAAAAAAAAAAAAAAAAAAAAAAAA

GreenieWarren committed:
No se preocupen chicos todo está bien

mi sanidad mental se ha recuperado

GreenieWarren committed:
LISTOO PAPAA, UPDATE CANONIZADO

YAYAYAJUUUUU

GreenieWarren committed:
Listo papá, a mimi por ahora

zzzzzzzz

Hector1567XD committed:
Change name of contact card

Hector1567XD committed:
Added useContactCardContext

Hector1567XD committed:
Merge branch 'feature/loaders' into feature/modal-contact

Hector1567XD committed:
Added fixes to Greenie ContactData form

Hector1567XD committed:
Added select max-height

Hector1567XD committed:
Merge branch 'feature/loaders' into feature/modal-contact

Nannajpg committed:
textfield sin bordes

Nannajpg committed:
cambios de estilos

Nannajpg committed:
estilos del modal

GreenieWarren committed:
Preparando contact Methods

Nannajpg committed:
estilos

Nannajpg committed:
Merge branch 'feature/modal-contact' of https://github.com/EquestriaReapers/ER-frontend into feature/modal-contact

Hector1567XD committed:
Merge branch 'main' into feature/modal-contact

Nannajpg committed:
estilos e idioma

Nannajpg committed:
Merge branch 'feature/modal-contact' of https://github.com/EquestriaReapers/ER-frontend into feature/modal-contact

Hector1567XD committed:
Added contactMethods to ContactCardContext to Greenie (ﾉ*ФωФ)ﾉ🎁

GreenieWarren committed:
transformando mi get a un post

Hello

Hector1567XD committed:
Added (!token) return in useAddContactMEthods

Nannajpg committed:
estilos

Hector1567XD committed:
Added selects languagues

Hector1567XD committed:
Merge branch 'feature/modal-contact' of https://github.com/EquestriaReapers/ER-frontend into feature/modal-contact

GreenieWarren committed:
Mostrar emails desde el back

GreenieWarren committed:
NO MORE MOCKS

Olvide hacerle commit a esto también hasdbheadas

Nannajpg committed:
estilos

GreenieWarren committed:
add y delete agregados

Todavía está el problema del id del metodo de contacto pero fuera de eso todo nice

Nannajpg committed:
boton disabled en agregar

luizzzito committed:
Correcciones de PR

luizzzito reviewed (approved):
Arregle cositas con relaciones a los estilos del boton, elimine comentarios innecesarios, y corregi errores ortograficos con respecto a nombres de variables.

Hector1567XD committed:
Merge branch 'main' into feature/modal-contact

Nannajpg committed:
margenes

Nannajpg committed:
fix

Nannajpg committed:
fix2

Hector1567XD committed:
Merge branch 'feature/modal-contact' of https://github.com/EquestriaReapers/ER-frontend into feature/modal-contact

Hector1567XD committed:
Added fixes and loading on add contact-method

Hector1567XD committed:
Merge branch 'main' into feature/modal-contact

Hector1567XD committed:
No me siento orgulloso de este codigo 😓

GreenieWarren committed:
Arreglando estilos de idioma

Hector1567XD committed:
Added fixes languagues

Hector1567XD committed:
Merge branch 'feature/modal-contact' of https://github.com/EquestriaReapers/ER-frontend into feature/modal-contact

GreenieWarren committed:
minor fixes de styles

Hector1567XD committed:
Added G reenie Cehins

GreenieWarren committed:
Merge branch 'feature/modal-contact' of https://github.com/EquestriaReapers/ER-frontend into feature/modal-contact

Hector1567XD committed:
Added updateContactDataAction

Hector1567XD committed:
Merge branch 'feature/modal-contact' of https://github.com/EquestriaReapers/ER-frontend into feature/modal-contact

Hector1567XD committed:
Succesful languagues functionality 🥂🎊

Hector1567XD commented:
Solo faltan cosas de responsive que se encargaran @GreenieWarren y @Nannajpg

GreenieWarren committed:
Me estresaba el botón de editicon

Cambie el tamaño de su cajita contenedora porque hacia que se hiciera mas grande la caja que contenia todo

GreenieWarren committed:
Mostrando los idiomas

GreenieWarren committed:
Quitando console.logs

GreenieWarren committed:
Poniendo el nivel

Hector1567XD committed:
Fixes reagrupation locations & languagues

GreenieWarren committed:
contact-modal responsive

hay... detallitos todavía

GreenieWarren committed:
Education responsive

(No creo que haya sido el mejor responsive, la otra solución requería mas trabajo)

Hector1567XD committed:
Merge branch 'main' into feature/modal-contact

Hector1567XD commented:
rEsponsive no es repsonsive
![image](https://github.com/EquestriaReapers/ER-frontend/assets/22270579/a309e6ab-fc14-4b5d-99dd-18e46e14642c)

Hector1567XD reviewed (changes requested):
maxHeight y un scroll

Hector1567XD reviewed (approved):
Ese fix que comente lo deje en https://github.com/EquestriaReapers/ER-frontend/pull/37

Nannajpg reviewed (approved):
Fino
